style.css

```diff
@@ -0,0 +1,106 @@
+@import "https://fonts.googleapis.com/css?family=Crete Round|Inter";
+
+header {
+  padding: 12px;
+}
```

Comment:
Good job working on the padding here.

[ Optional Changes ] All changes marked optional won't stop your program from being approved but will help make your program better.

- [ ] Kindly add the code below above the `header` style to reset all styles on the entire page to take off the white margin on all corners of the website in order for your design to match the Figma template. 
```mdx 
* {
  margin: 0;
  padding: 0;
}
```

**Your design**
![image](https://user-images.githubusercontent.com/50941074/164923399-cde7847b-5c5b-4f15-9357-24423e9b4a5d.png)

**The Figma Design**
![image](https://user-images.githubusercontent.com/50941074/164924284-05b22666-5839-4200-bc00-149228305c38.png)